artifacts/AdditionalSetup.ps1

```diff
@@ -327,6 +327,19 @@ if (($env:cosmoServiceRestart -eq $false) -and ![string]::IsNullOrEmpty($env:saa
     Set-NAVServerInstance -ServerInstance $ServerInstance -Restart
 }
 
+
+Invoke-LogEvent -name "Import K8S Helper" -telemetryClient $telemetryClient
+Write-Host "Import K8S Helper"
```

Comment:
Do we need to load the module in the additional setup? Isn't it enough to load it when calling it from the API? When executing a command from the API there will be an "empty" PowerShell session anyway.

Reply:
yes it's possible, i've removed the import